src/main/java/org/gridsuite/gateway/filters/TokenValidatorGlobalPreFilter.java

```diff
@@ -125,8 +130,18 @@ public Mono<Void> filter(ServerWebExchange exchange, GatewayFilterChain chain) {
             exchange.getRequest()
                 .mutate()
                 .headers(h -> h.set(HEADER_USER_ID, jwtClaimsSet.getSubject()));
-        } catch (JOSEException | BadJOSEException | ParseException | MalformedURLException e) {
-            LOGGER.info("{}: 401 Unauthorized, The token cannot be trusted : {}", exchange.getRequest().getPath(), e.getMessage());
+        } catch (JOSEException | BadJOSEException err) {
+            try {
+                cacheService.evictSingleCacheValue("JwksUrl", jwt.getJWTClaimsSet().getIssuer());
```

Comment:
Add a static constant for the cache name

Reply:
done